### PR TITLE
RSDK-2830 - add servo documentation

### PR DESCRIPTION
**Pull request by clintpurser**
Add documentation comments to servo.dart.

I got these comments from the python sdk [servo.py](https://github.com/viamrobotics/viam-python-sdk/blob/main/src/viam/components/servo/servo.py)

### CODE REVIEW

File: lib/src/components/servo/servo.dart [hunk 1]
```diff
@@ -11,12 +15,25 @@ abstract class Servo extends Resource {
...
+  ///     int: The current angle of the servo in degrees.
   Future<int> getPosition({Map<String, dynamic>? extra});
 
+  /// Stop the [Servo]. It is assumed that the servo stops immediately.
   Future<void> stop({Map<String, dynamic>? extra});
 
+  /// Get if the [Servo] is currently moving.
+  ///
+  /// Returns:
+  ///     bool: Whether the servo is moving
```
Comment (maximpertsov) [inline lib/src/components/servo/servo.dart:36]: [nit] consistency

```suggestion
  ///     bool: Whether the servo is moving.
```

**Review by maximpertsov — DISMISSED** [review]

File: lib/src/components/servo/servo.dart (hunk 1 above)
Comment (clintpurser) [reply lib/src/components/servo/servo.dart]: Good catch, thank you!

**Review by cheukt — APPROVED** [review]

File: lib/src/components/servo/servo.dart [hunk 2]
```diff
@@ -11,12 +15,25 @@ abstract class Servo extends Resource {
 
   Servo(this.name);
 
+  /// Move the [Servo] to the provided angle.
+  ///
+  /// Args:
+  ///     [angle] (int): The desired angle of the servo in degrees.
```
Comment (njooma) [inline lib/src/components/servo/servo.dart:21]: Similar to my [comment on motors](https://github.com/viamrobotics/viam-flutter-sdk/pull/10#discussion_r1184352469), `Args` and `Returns` should be removed

**Review by njooma — CHANGES_REQUESTED** [review]

File: lib/src/components/servo/servo.dart (hunk 2 above)
Comment (clintpurser) [reply lib/src/components/servo/servo.dart]: ✅ thanks for calling this out

**Review by njooma — APPROVED** [review]

**Review by cheukt — APPROVED** [review]

**Pull request merged**